### PR TITLE
add go-guru-definition-other-window

### LINKED ISSUE #175: add go-guru-definition equivalent of godef-jump-other-window
It'd be nice to only use guru in my emacs sessions, but there's no obvious equivalent to `godef-jump-other-window` in the `go-guru` functions.

`godef-jump-other-window` passes along the `other-window` variable to code I don't understand well enough to make equivalent hacks for `go-guru-definition` for. 

Maybe there's something obvious that I missed either in that code or in how to open up another window with the current `go-guru` functions.

### PR DESCRIPTION
**Pull request by jmhodges**
Adds go-guru-definition-other-window that opens the defintion of the
target in another window. Equivalent to godef-jump-other-window.

Fixes #175

### CODE REVIEW

File: go-guru.el [hunk 1]
```diff
@@ -319,11 +319,11 @@ If BUFFER, return the number of characters in that buffer instead."
   "Go to the OFFSETth byte in the current line."
   (goto-char (byte-to-position (+ (position-bytes (point-at-bol)) (1- offset)))))
 
-(defun go-guru--goto-pos (posn)
+(defun go-guru--goto-pos (posn other-window)
```
Comment (syohex) [inline go-guru.el]: Is passing function better than passing flag value ? If its argument is function, we can pass other `find-file` function like `find-function-other-frame` etc.

Comment (jmhodges) [inline go-guru.el]: @dominikh what's your opinion?

**Comment by jmhodges** [comment]
Ping!

**Comment by dominikh** [comment]
Sorry, I'll look into this in the next couple of days.

**Comment by jmhodges** [comment]
Ping

**Comment by dominikh** [comment]
Hey, sorry for the delay. I'll look at it as soon as I can, though I'm not sure when that'll be :-(

**Comment by jmhodges** [comment]
Gentle ping!

File: go-guru.el (hunk 1 above)
Comment (dominikh) [inline go-guru.el]: For now: KISS.

**Comment by dominikh** [comment]
Merged as 3f02a165dba38295ae4b961482fe50e012a2112a, thank you.

**Comment by jmhodges** [comment]
Thanks!

**Pull request closed without merging**